projects/vs/psa-boot/psa-boot.vcxproj.user

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@asenaasena @Oruccu could you please remove this file and commit and push again. Thanks